plugins/tokenizers/mecab.c

```diff
@@ -110,14 +113,19 @@ grn_mecab_init(grn_ctx *ctx,
                grn_mecab *mecab,
                const char *tag)
 {
+  mecab->mecab_model = NULL;
   mecab->mecab = NULL;
-  mecab->mutex = grn_plugin_mutex_open(ctx);
```

Comment:
We still always needs the mutex for `mecab_init_mecab()`.
We should avoid creating `mecab_model_t *` in multiple threads.